lib/common.sh

```diff
@@ -136,7 +136,7 @@ export BMO_BASE_URL="${BMO_BASE_URL:-metal3-io/baremetal-operator}"
 export RUN_LOCAL_IRONIC_SCRIPT="${BMOPATH}/tools/run_local_ironic.sh"
 
 export CAPM3PATH="${CAPM3PATH:-${M3PATH}/cluster-api-provider-metal3}"
-export CAPM3_BASE_URL="${CAPM3_BASE_URL:-metal3-io/cluster-api-provider-metal3}"
+export CAPM3_BASE_URL="Nordix/cluster-api-provider-metal3"
```

Comment:
Tip: It is much cleaner to put the wanted test configuration in `config_example.sh` than change them all over the place in `lib`.

Reply:
its a test commit. Thanks for the tip .